atlp-rwanda/ecommerce-app-predators-bn
# #27 feature: recommend product to a buyer

**What does this PR do?**

recommend products to the buyer

**Description of Task to be completed**


this PR will take the category_id of search products and return other products of the same category as recommended products 

**How should this be manually tested?**
clone this directory

1. open your terminal in Vscode 
- run `git clone https://github.com/atlp-rwanda/ecommerce-app-predators-bn.git`
- run `git checkout ft-recommendProduct-27`
- run `npm install`
- run `npm run dev`

2. open postman and past the following:

-**example1:** past: http://localhost:3000/api/product/all/search?keyword=google pixel

-**example2:** past: http://localhost:3000/api/product/all/search?keyword=iPhone

What are the relevant pivotal tracker stories?
[-card#27](https://trello.com/c/zJJHCwqi/27-buyer-should-be-able-to-see-recommended-products)
